### DISCUSSION TITLE
Moving the .autoortho-data folder on Windows

### DISCUSSION THREAD

**Discussion by fabian0hofmann** [kubilus1/autoortho#235] [Q&A]
Hi, I was just trying to install autoortho but I am extremely storage restricted on my main hard drive. Hence I can't install it properly because I don't have enough storage. So I was wondering whether there's a way to move the folder elsewhere, either the entire directory or by shortcut. Sorry if this is a trivial question, I'm not really familiar with autoortho yet but as far as I can tell, nobody has asked this before.
Thanks for your help.

**Comment by dpdurst** [comment]
AO 5.0.x will allow you to assign the Cache, Download and Scenery folder to another drive. Please be a little more specific on installation of XP, AO and version of AO as it may be possible

**Reply by fabian0hofmann** [reply]
I'm using the most current XP11 version (11.55) and the autoortho version which was available four days ago (says 0.4.4 in releases?).

**Comment by kubilus1** [comment]
You can install components in different locations.  The 'Custom Scenery'
location must be a local NTFS drive, however.  The cache and download dir
can be pretty much anywhere, but keep in mind storage speed with the cache
drive.

XPlane allows you to make shortcuts for custom scenery, so just follow
standard scenery setup procedure.  For instance shortcut the `z_ao_na`
package into the X Plane custom scenery dir.

**Reply by fabian0hofmann** [reply]
What do you mean by 'Custom Scenery' location? My XP is not installed on what I think is my NTFS drive (I assume that's "where my windows lives") so my 'Custom Scenery' isn't on the NTFS drive. What I'm trying to move is the folder autoortho installs its stuff into when I open autoortho, go to 'Scenery' and try to install the ortho sets. Is that what you call 'Custom Scenery' or can I move that folder? And if I can move it, is there any documentation or tutorial specifying how it works exactly?

**Comment by dpdurst** [comment]
See this, If you are using 4.4 then you may want to remove that and download the 5.x version. The 5.x version allows you to specify where the temp download folder is located, the cache folder and your scenery downloads folder for xplane if you want to store those scenery files someplace else.

Doc link - https://github.com/kubilus1/autoortho/discussions/238

**Reply by fabian0hofmann** [reply]
alright thanks, that seems to have got it working. Windows Defender flagged the .exe as a trojan on download though... I guess that's a misflag?

**Reply by dpdurst** [reply]
Yes it is, I've had to setup exceptions in Defender for the folder where AO resides so it stopped doing that. Once you do that you'll be fine
